reana-snakemake.yaml

```diff
@@ -0,0 +1,28 @@
+version: 0.8.0
```

Comment:
We could name this file `reana.yaml` so that it would be taken by default when someone does `reana-client run`. This would be more comfortable to the users. We don't have non-Snakemake versions of the workflow, so this would do.  (And, should we have some non-Snakemake workflow equivalents, we can always rename the file later.)